guide/easy_install.md

```diff
@@ -0,0 +1,156 @@
+# Installing Moonfire NVR using provided scripts
+
+There are no binary packages of Moonfire NVR available yet, so it must be built
+from source. This is made easy using a few scripts that will do the job for you
+unless you have very a different operating system. The scripts are written and
+tested under ubuntu and raspbian but should not be hard to modify if necessary.
+You'll start by downloading moonfire if you have not already done so.
+
+## Downloading
+
+See the [github page](https://github.com/scottlamb/moonfire-nvr) (in case
+you're not reading this text there already). You can download the bleeding
+edge version from the command line via git:
+
+    $ git clone https://github.com/scottlamb/moonfire-nvr.git
+
+## Preparation steps for easy install
+
+There are a few things to prepare if you want a truly turnkey install, but
+they are both optional.
+
+### Cameras.sql preparation
```

Comment:
I think now that there's "moonfire-nvr config", it's much easier to go through those menus than prepare this file. I tried to remove all references to it when adding that command, but I see now I missed one in .gitignore and one in guide/install.md.